std_msgs/Header header
int32 dog_state